workflow.rst

```diff
@@ -158,13 +158,20 @@ As configured, the following property is used by the marking store::
 
     The marking store type could be "multiple_state" or "single_state". A single
     state marking store does not support a model being on multiple places at the
-    same time.
+    same time. This means a "workflow" must use a "multiple_state" marking store
+    and a "state_machine" must use a "single_state" marking store. Symfony
+    configures the marking store according to the "type" by default, so it's
+    preferable to not configure it.
+
+    A single state marking store use a string to store the data. A multiple
+    state marking store use an array to store the data.
 
 .. tip::
 
-    The ``type`` (default value ``single_state``) and ``arguments`` (default
```

Comment:
I'm not  convinced with this tip + previous note.
There is too much to way about the marking store.
Chould we add a new chapter (in this page OC) instead?